cygnus-ngsi/src/main/java/com/telefonica/iot/cygnus/sinks/NGSIHDFSSink.java

```diff
@@ -1,6 +1,6 @@
 /**
  * Copyright 2014-2017 Telefonica Investigación y Desarrollo, S.A.U
- *
+ *purposes
```

Comment:
The addition of this line seems to be a typo...

Reply:
Fixed in [c434ab4](https://github.com/telefonicaid/fiware-cygnus/pull/1799/commits/c434ab4ded7d5c6ba56544524927822c68aeaa4b)